pkg/netpol/diff/diff_errors.go

```diff
@@ -2,7 +2,9 @@ package diff
 
 // diffGeneratingError - DiffError that may arise while producing the connectivity diff report
 type diffGeneratingError struct {
```

Comment:
why do the functions `func (e *diffGeneratingError) IsFatal() ` and `func (e *diffGeneratingError) IsSevere()` return constant values and not depend on the values of `fatal, severe` members of `diffGeneratingError` ?